doc/src/manual/cowboy_rest.asciidoc

```diff
@@ -99,10 +103,10 @@ For PUT requests, the body is a representation of the resource
 that is being created or replaced.
 
 For POST requests, the body is typically application-specific
-instructions on how to process the request, but it may also
-be a representation of the resource. When creating a new
-resource with POST at a different location, return `{true, URI}`
-with `URI` the new location.
+instructions on how to process the request, but it may also be a
+representation of the resource. When creating a new resource with POST
+at a different location, return `{create, URI}` or `{see_other, URI`}
+with `URI` the new location. The return value `{true, URI}` is deprecated.
```

Comment:
If it's deprecated then it should no longer be documented. Also the deprecation should be indicated in the changelog at the bottom of the file instead.

Reply:
Ok.  Which version number should I use?  2.9?

Reply:
Yes.